static/app/views/dashboardsV2/manage/index.tsx

```diff
@@ -0,0 +1,154 @@
+
+import Feature from 'app/components/acl/feature';
+import Alert from 'app/components/alert';
+import AsyncComponent from 'app/components/asyncComponent';
+import Breadcrumbs from 'app/components/breadcrumbs';
+import LightWeightNoProjectMessage from 'app/components/lightWeightNoProjectMessage';
+import PageHeading from 'app/components/pageHeading';
+import SearchBar from 'app/components/searchBar';
+import SentryDocumentTitle from 'app/components/sentryDocumentTitle';
+import {t} from 'app/locale';
+import {PageContent, PageHeader} from 'app/styles/organization';
+import space from 'app/styles/space';
+import {Organization} from 'app/types';
+import withOrganization from 'app/utils/withOrganization';
+
+import {DashboardListItem} from '../types';
+
+import DashboardList from './dashboardList';
+
+type Props = {
+  organization: Organization;
+  location: Location;
+  router: ReactRouter.InjectedRouter;
+} & AsyncComponent['props'];
+
+type State = {
+  dashboards: DashboardListItem[] | null;
+  dashboardsPageLinks: string;
+} & AsyncComponent['state'];
+
+class ManageDashboards extends AsyncComponent<Props, State> {
+  getEndpoints(): ReturnType<AsyncComponent['getEndpoints']> {
+    const {organization, location} = this.props;
+    return [
+      [
+        'dashboards',
+        `/organizations/${organization.slug}/dashboards/`,
+        {
+          query: {
+            ...pick(location.query, ['cursor', 'query']),
+            per_page: '9',
+          },
+        },
+      ],
+    ];
+  }
+
+  handleSearch = (query: string) => {
+    const {location, router} = this.props;
+
+    router.push({
+      pathname: location.pathname,
+      query: {...location.query, cursor: undefined, query},
+    });
+  };
+
+  getQuery() {
+    const {query} = this.props.location.query;
+
+    return typeof query === 'string' ? query : undefined;
+  }
+
+  renderActions() {
+    return (
+      <StyledActions>
+        <StyledSearchBar
+          defaultQuery=""
+          query={this.getQuery()}
+          placeholder={t('Search Dashboards')}
+          onSearch={this.handleSearch}
+        />
+      </StyledActions>
+    );
+  }
+
+  renderNoAccess() {
+    return (
+      <PageContent>
+        <Alert type="warning">{t("You don't have access to this feature")}</Alert>
+      </PageContent>
+    );
+  }
+
+  renderBody() {
+    const {dashboards, dashboardsPageLinks} = this.state;
+    const {organization, location} = this.props;
+    return (
+      <DashboardList
+        dashboards={dashboards}
+        organization={organization}
+        pageLinks={dashboardsPageLinks}
+        location={location}
+      />
+    );
+  }
+
+  render() {
+    const {organization} = this.props;
+
+    return (
+      <Feature
+        organization={organization}
+        features={['dashboards-manage']}
+        renderDisabled={this.renderNoAccess}
+      >
+        <SentryDocumentTitle title={t('Manage Dashboards')} orgSlug={organization.slug}>
```

Comment:
maybe instead of AsyncComponent, you could use AsyncView. Using the AsyncView component you can set the page's title in the `getTitle` function